components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Publisher/src/Services/PublisherJobService.cs

```diff
@@ -335,8 +342,7 @@ public sealed class PublisherJobService : IPublishServices<string> {
             List<PublishedDataSetVariableModel> variables = null;
             foreach (var writer in publishJob.WriterGroup.DataSetWriters) {
                 if (writer.DataSet.DataSetSource.Connection.IsSameAs(connection) &&
-                    writer.DataSet.DataSetSource.SubscriptionSettings?.PublishingInterval ==
```

Comment:
Why remove this?

Reply:
the dataSetWriterId is now EndpointId+samplingInterval, so the sampling interval comparison was replaced with dataSetWriterId comparison. It would had been redundant to keep it.